src/dashboards/teachers/details.jsonnet

```
@@ -398,3 +398,415 @@ dashboard.new(
+      },
+      metrics=[common.metrics.count],
+      bucketAggs=[
+        {
+          id: '5',
+          field: common.fields.actor_account_name,
+          settings: {
+            min_doc_count: '1',
+            size: '0',
+            order: 'desc',
+            orderBy: '_count',
+          },
+          type: 'terms',
+        },
+      ],
+      timeField='@timestamp'
+    )
+  ),
+  gridPos={ x: 0, y: 26, w: 4, h: 6 }
+).addPanel(
+  {
+    type: 'barchart',
+    title: 'Subtitle language user choice',
+    options: {
+      legend: {
+        displayMode: 'hidden',
+      },
+    },
+    fieldConfig: {
+      overrides: [
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Unique Count',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Users',
+            },
+          ],
+        },
+      ],
+    },
+    description: |||
+      The distribution of subtitle languages chosen by users when it is activated.
+    |||,
+    targets: [
+      {
+        datasource: common.datasources.lrs,
+        query: '%(video_query)s AND verb.id:"interacted"' % {
+          video_query: teachers_common.queries.video_id,
+        },
+        metrics: [common.metrics.cardinality(common.fields.actor_account_name)],
+        bucketAggs: [
+          {
+            id: '2',
+            type: 'terms',
+            settings: {
+              min_doc_count: '1',
+              size: '10',
+              order: 'desc',
+              orderBy: '_term',
+            },
+            field: common.fields.subtitle_language,
+          },
+        ],
+        timeField: '@timestamp',
+      },
+    ],
+    datasource: common.datasources.lrs,
+  },
+  gridPos={ x: 4, y: 26, w: 4, h: 6 }
+)
+.addPanel(
+  statPanel.new(
+    title='Full screen activation',
+    description=|||
+      The number of learners who have switched to full screen.
+    |||,
+    datasource=common.datasources.lrs,
+    fields=common.fields.actor_account_name,
+    graphMode='none',
+    reducerFunction='count',
+    unit='none',
+  ).addTarget(
+    elasticsearch.target(
+      datasource=common.datasources.lrs,
+      query='%(video_query)s AND verb.id:"interacted" AND %(full_screen)s:true' % {
+        video_query: teachers_common.queries.video_id,
+        full_screen: common.utils.single_escape_string(common.fields.full_screen),
+      },
+      metrics=[common.metrics.count],
+      bucketAggs=[
+        {
+          id: '5',
+          field: common.fields.actor_account_name,
+          settings: {
+            min_doc_count: '1',
+            size: '0',
+            order: 'desc',
+            orderBy: '_count',
+          },
+          type: 'terms',
+        },
+      ],
+      timeField='@timestamp'
+    )
+  ),
+  gridPos={ x: 8, y: 26, w: 4, h: 6 }
+).addPanel(
+  {
+    type: 'barchart',
+    title: 'Video speed user choice',
```

Not a big fan of "user choice", what about the following?

```suggestion
    title: 'Changed video speed',
```

---

Same remarks for the following panel titles.